src/Razor/test/Microsoft.AspNetCore.Razor.Test.ComponentShim/Microsoft.AspNetCore.Components/EventCallbackFactory.cs

```diff
@@ -16,12 +18,26 @@ public sealed class EventCallbackFactory
 
         public EventCallback Create(object receiver, Func<object, Task> callback) => default;
 
+        public EventCallback<T> Create<T>(object receiver, string callback) => default;
```

Comment:
This is needed for event handlers. When we see an attribute being set on an element, we can't tell whether you're passing in a delegate or a string (JS).